Library/Formula/uberftp.rb

```diff
@@ -3,24 +3,33 @@
 class Uberftp < Formula
   desc "Interactive GridFTP client"
   homepage 'http://dims.ncsa.illinois.edu/set/uberftp/'
-  url 'https://github.com/JasonAlt/UberFTP/archive/Version_2_7.tar.gz'
-  sha1 'f185e2ed567eca3484ca230e44a6ffdb4ec69792'
+  url 'https://github.com/JasonAlt/UberFTP/archive/Version_2_8.tar.gz'
+  sha256 '8a397d6ef02bb714bb0cbdb259819fc2311f5d36231783cd520d606c97759c2a'
 
   depends_on 'globus-toolkit'
 
   def install
-    # get the flavor
     globus = Formula["globus-toolkit"].opt_prefix
+    globus_version = Formula["globus-toolkit"].version
 
-    core = `"#{globus}/sbin/gpt-query" globus_core`
-    flavor = case core
-      when /gcc64dbg/ then "gcc64dbg"
-      when /gcc32dbg/ then "gcc32dbg"
+    if globus_version.to_s[0].to_i >= 6
```

Comment:
If somebody pinned the globus version to an older version, he can't compile uberftp anymore. However, if that is not a use case I can drop the ```if``` condition here.